Manifests/ManifestsApple/com.apple.dnsProxy.managed.plist

```diff
@@ -183,8 +182,6 @@ The payload organization for a payload need not match the payload organization i
 			<string>plist2dict</string>
 		</dict>
 	</array>
-	<key>pfm_supervised</key>
```

Comment:
This line from the documentation doesn't make sense to me:

"Beginning with iOS 15, this profile is unsupervised and must be installed via MDM."

Reply:
Yeah it should have said _Beginning with iOS 15, this payload no longer requires a supervised device but must be installed via MDM_.